zinc/src/main/scala/sbt/internal/inc/IncrementalCompilerImpl.scala

```diff
@@ -375,7 +375,7 @@ class IncrementalCompilerImpl extends IncrementalCompiler {
       if (outputJarsOnCp) sys.props.put("scala.classpath.closeZip", "true")
 
       val extraScalacOptions = {
-        if (outputJarsOnCp && scalaCompiler.scalaInstance.version.startsWith("2.12"))
+        if (outputJarsOnCp && !Set("2.10", "2.11")(scalaCompiler.scalaInstance.version.take(4)))
```

Comment:
How can we be sure about what flags future Scala 3.x would support?

Reply:
Most likely it doesn't.  But I was thinking about a potential 2.14.